docs/how_to_add_packages.md

```diff
@@ -154,10 +155,9 @@ Also, **every `conanfile.py` should be accompanied by one or several folder to t
 ### The test package folders: `test_package` and `test_<something>`
 
 All the packages in this repository need to be tested before they join ConanCenter. A `test_package` folder with its corresponding `conanfile.py` and
-a minimal project to test the package is strictly required. You can read about it in the
+a minimal project to test the package is strictly required. You can read about it in the [Conan documentation](https://docs.conan.io/en/latest/creating_packages/getting_started.html#the-test-package-folder).
 
-# FIXME: This link no longet exist and there is no dedicated section about test package in docs.
```

Comment:
I added that FIXME to be replaced before merging that PR, but as you can see, I forgot to remove 😅

Reply:
I just wanted to make it less obvious 🙈 

I have 👇 which will replace this in the follow up PRs 🤞 

https://github.com/prince-chrismc/conan-center-index/blob/docs-overhaul/docs/adding_packages/test_packages.md